utils/capellacontrol/controller.go

```diff
@@ -1096,6 +1096,32 @@ func (c *Controller) DisablePrivateEndpoints(
 	return nil
 }
 
+func (c *Controller) EnablePrivateEndpointsColumnar(
+	ctx context.Context,
+	tenantID, projectID, clusterID string,
+) error {
+	path := fmt.Sprintf("/v2/organizations/%s/projects/%s/instance/%s/privateendpoint", tenantID, projectID, clusterID)
```

Comment:
Should tenantID, projectID, and clusterID be URL-encoded before they are slotted into the HTTP path?

Ditto other places variables are used in paths.

Reply:
True probably yes, though the all these id's are of a standard format and _shouldnt_ need it. And I was more copying the style of these functions already there. 

Maybe I can do a follow up patch after to change this for all the functions so they're consistent?